client/scripts/stores/ActivityStore.ts

```diff
@@ -0,0 +1,110 @@
+import { modelFromServer } from '../controllers/server/threads';
```

Comment:
This is...more complicated than a store, it's two stores attached to each other. Feels like this should be a controller.

Or we can leave this as is but I would leave a comment if we do that.

Reply:
I'll leave a comment for now, but more than happy to split it up into two Stores in a single file if you think it'd be more organizationally clear! I linked them up because they're linked to a single controller (`recent_activity.ts`(

Reply:
I think two stores in one file would make sense! Just would be nice to have a clear distinction between what is a store vs. a controller / prevent stores from containing business logic.